azure-iot-hub-extractor/example_config.yaml

```diff
@@ -0,0 +1,31 @@
+version: "1"
+
+logger:
+    console:
+        level: INFO
+
+cognite:
+    # Read these from environment variables
+    host: ${COGNITE_BASE_URL}
+    api-key: ${COGNITE_API_KEY}
+    project: ${COGNITE_PROJECT}
+
+metrics:
+    cognite:
+        external_id_prefix: "iot_metrics"
+        asset_name: "Azure IOT Hub Extractor metrics"
+        asset_external_id: "iot_metrics"
```

Comment:
Is this necessary for the example config?

Reply:
Commented it out. Nice to have a sample somewhere?

Reply:
True, but this is not the recommended metrics setup, and the example file is often interpreted as 'best practice' - which I fear can lead to some confusion.